pr_check.sh

```diff
@@ -17,23 +17,27 @@
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
-COMPONENT_NAME="ccx-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
-IMAGE="quay.io/cloudservices/ccx-smart-proxy"
+export APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ccx-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```

Comment:
I think this should be `insights-results-smart-proxy`

Reply:
'ccx-smart-proxy' is the old name from saas.yml file.

Reply:
"ccx-smart-proxy" is also the name of the [ClowdApp](https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/deploy/clowdapp.yaml#L40)